### ISSUE TITLE
Any way to replicate my ESO inventory with Arkinventory?

### ISSUE THREAD

**Issue by Myrthian** [arkayenro/arkinventory#1414]
I've been mostly playing ESO for the last years using similar addons to Arkinventory as a means of sorting + grouping my inventory.

Is there a way to replicate how my ESO inventory looks + functions with horizontal groups with the tab/group names visible on top?
![00](https://user-images.githubusercontent.com/30933093/102336137-12041f80-3f91-11eb-8ff6-d41e176725f6.png)

**Comment by arkayenro** [comment]
you can set the number of bars in each row, what categories go in each bar, sort each bar individually if you want, and name them as well, but i dont do tabs, just one big scrollable bag

i think one of the other bag mods does tabs though

**Comment by Myrthian** [comment]
After playing around with it for a while longer I'm very close with 2 questions remaining:
- Is there a way to have the bag width be adjusted to the row with the most entries and not have dead space in there?
- Is there a way to force displaying the full name of a row despite it only having 1 entry? The "show empty" option under Bars - Styles - General seemingly does parts of it

![00](https://user-images.githubusercontent.com/30933093/102507483-9a172180-4084-11eb-955d-f577995c5642.png)

**Comment by dorana** [comment]
Yes, well kinda. if you right click on a group in Edit mode and set the minimum width it will lock to always show it at a width of X items, or more

**Comment by arkayenro** [comment]
the window width is fixed - it will expand based on having all fixed width bars but no ones ever asked for it to shrink back to whatever the bars are doing.  ill check if thats possible and add an option if it is

you may be better off setting the widow width to say 10, bars per row to 1 (as you have), then set the global min bar width to 10 (same as the width) so that the bars stretch that long and the name will fit
